manifests/web.pp

```diff
@@ -151,6 +151,9 @@
 # [*ldap_clientkey*]
 # Set location of client key used by LDAP authentication.
 #
+# [*ldap_reqcert *]
+# Specifies what checks to perform on a server certificate ( never | allow | try | demand,hard )
```

Comment:
The `demand,hard` is a bit odd. However, since there's a data type I'd suggest to leave them out. If you generate a `REFERENCE.md` the data type should be enough.

Reply:
updated it. I checked the docs, `demand` and `hard` are independent values.